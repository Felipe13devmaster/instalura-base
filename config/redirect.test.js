import redirects from './redirects';

describe('configs/redirects', () => {
  test('renderizar todos os redirects atuais', () => {
    expect(redirects).toMatchSnapshot();
    // tira uma foto do snapshot e compara automaticamente com a vers√£o anterior.
  });
});
