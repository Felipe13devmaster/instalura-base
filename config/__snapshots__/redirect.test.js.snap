// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`configs/redirects renderizar todos os redirects atuais 1`] = `
Array [
  Object {
    "destination": "/app/login/",
    "permanent": true,
    "source": "/login/",
  },
]
`;
